src/server/worker.cc

```diff
@@ -38,6 +38,8 @@
 #include <openssl/ssl.h>
 #endif
 
+#include <oneapi/tbb/parallel_for.h>
+#include <oneapi/tbb/parallel_reduce.h>
```

Comment:
nit: \n between 3rd and sys

Reply:
Hi @mapleFU ,

Thanks for your suggestions!

I find the formatter will automatically remove the blank line.
So, I make the tbb headers close to above glog headers.

Best Regards,
Edward